src/content/blog/unleashing-php-development-with-ddev.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Unleashing PHP Development with DDEV: The Power of Upstream Support"
+title: "Why Our Agency Became a Major Supporter of DDEV ... and Yours Should Too"
```

Comment:
I’m late @rfay, but dropping the `...` would strengthen this further. (For me `...` amounts to an unnecessary pause and visual interruption.)